doc/api/fs.md

```diff
@@ -3484,7 +3484,7 @@ added: v19.8.0
 * `path` {string|Buffer|URL}
 * `options` {Object}
   * `type` {string} An optional mime type for the blob.
-* Return: {Promise} containing {Blob}
+* Returns: {Promise} Fulfills with a {Blob} upon success.
```

Comment:
```suggestion
* Returns: {Promise} Resolves with a {Blob} upon success.
```

Reply:
Fulfills is the correct term here, see https://tc39.es/ecma262/#sec-properties-of-promise-instances. IIUC the nuance is that promise resolution can end up either fulfilling the promise or rejecting it (e.g. `Promise.resolve(Promise.reject(null))`, `Promise.resolve` creates a `Promise` that resolves to the value of `Promise.reject(null)`, but that promise gets rejected).

Reply:
@aduh95 thanks for clarifying